src/data/defaultRecipes.js

```diff
@@ -0,0 +1,3 @@
+const allRecipes = []
+
+export default allRecipes;
```

Comment:
Ovo možeš da uzmeš moje default recipes tako što ćeš mergovati master u ovaj branch

Reply:
svejedno